src/test/addresses_goerli.sol

```diff
@@ -311,6 +311,17 @@ contract Addresses {
         addr["RWA009_A_URN"]                    = 0xd334bbA9172a6F615Be93d194d1322148fb5222e;
         addr["RWA009_A_JAR"]                    = 0xad4e1696d008A656F810498A974C5D3dC4A6150d;
         addr["RWA009_A_OUTPUT_CONDUIT"]         = 0x5DCdbD3cCF9B09EAAD03bc5f50fA2B3d3ACA0121;
+        addr["RWA007"]                          = 0x02091C59C92fc5E0575b3B133c3caa4f57D79b0e;
```

Comment:
the ordering here is inconsistent with the one followed below in the `config.sol`

Reply:
Done